src/category_theory/opposites.lean

```diff
@@ -222,6 +222,8 @@ nat_iso.of_components (λ X, iso.refl _) (by tidy)
 def right_op_left_op_iso (F : Cᵒᵖ ⥤ D) : F.right_op.left_op ≅ F :=
 nat_iso.of_components (λ X, iso.refl _) (by tidy)
 
+lemma right_op_left_op_eq (F : Cᵒᵖ ⥤ D) : F.right_op.left_op = F := by { cases F, refl, }
```

Comment:
I don't think this is used? I think the mathlib consensus is to not state equalities of functors as they cause trouble when misused.

Reply:
It is used in the proof of `strict_universal_property_fixed_target.op`. I do not think this is a misuse of equalities of functors (which are not necessarily evil). It is occasionnaly needed in the internals of the API of the localization of categories, in particular for the strict universal property of the localization (whose field `fac` involves equalities of functors!). There is no alternative here...

Reply:
I see. In this case, I think there should be a warning in the docstring and/or a pointer to the natural isomorphism version.

Reply:
Done.